hermes-consumers/src/test/groovy/pl/allegro/tech/hermes/consumers/consumer/receiver/SchemaExistanceEnsurerTest.groovy

```diff
@@ -0,0 +1,127 @@
+package pl.allegro.tech.hermes.consumers.consumer.receiver
```

Comment:
Typo in filename, should be `SchemaExistenceEnsurerTest` not `SchemaExistanceEnsurerTest `